src/Components/Components/src/ComponentBase.cs

```diff
@@ -41,6 +42,29 @@ public ComponentBase()
         };
     }
 
+    /// <summary>
+    /// Gets the <see cref="ComponentPlatform"/> the component is running on.
+    /// </summary>
+    protected ComponentPlatform Platform => _renderHandle.Platform;
+
+    /// <summary>
+    /// Gets the target <see cref="IComponentRenderMode"/> this component will run in after prerendering.
```

Comment:
```suggestion
    /// Gets the <see cref="IComponentRenderMode"/> assigned to this component.
```

I think the phrasing shouldn't assume we are currently prerendering, or that prerendering is even applicable (e.g., in standalone WebAssembly apps, it wouldn't make sense).